src/sugar.rs

```diff
@@ -1071,3 +1089,17 @@ mod ownership_tests {
         }
     }
 }
+
+#[cfg(test)]
+mod closure_tests {
+    #[test]
+    fn test_simple() {
+        let x = 42;
+
+        let _ = proptest! {
+            |(y in 0..100)| {
+                assert!(x != y);
+            }
+        };
```

Comment:
... add a test that ensures that it keeps working if we make changes :)

Reply:
Thanks, fixed. This one slipped my mind!